fuel/transformers/__init__.py

```diff
@@ -180,15 +300,20 @@ class ScaleAndShift(SingleMapping):
     def __init__(self, data_stream, scale, shift, **kwargs):
         self.scale = scale
         self.shift = shift
-        super(ScaleAndShift, self).__init__(data_stream, **kwargs)
+        kwargs.setdefault('axis_labels', data_stream.axis_labels)
```

Comment:
Is there a case where we want `axis_labels` to be overrideable?

The only case where I can think of that we would want to override is if scale and shift are arrays, and there could be some broadcasting going on. But right now the assumption is that they are scalars...